Source/scrollrt.cpp

```diff
@@ -199,6 +216,20 @@ static void scrollrt_draw_cursor_item()
 	my++;
 	gpBufEnd = &gpBuffer[BUFFER_WIDTH * (SCREEN_HEIGHT + SCREEN_Y) - cursW - 2];
 
+	int ckey = 255;
+	int pass = 0;
+
+blit:
+	pass++;

```

Comment:
Can you elaborate why we do have a "multi-pass" process in here? If at all possible, try documenting it by making changes to the code to make it more clear what is happening.